.github/workflows/agp-matrix.yml

```diff
@@ -73,6 +63,7 @@ jobs:
           disable-animations: true
           disable-spellchecker: true
           target: 'aosp_atd'
+          arch: x86
```

Comment:
any specific reason we need `x86` here?

Reply:
yeah, it's faster 👀 or actually that is the default anyway so we could remove it https://github.com/ReactiveCircus/android-emulator-runner#configurations